cabextract.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: cabextract
+  version: 1.11
+  epoch: 0
+  description: "Tool for extracting Microsoft cabinet files"
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libmspack-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://www.cabextract.org.uk/cabextract-${{package.version}}.tar.gz
+      expected-sha256: b5546db1155e4c718ff3d4b278573604f30dd64c3c5bfd4657cd089b823a3ac6
+
+  - name: Configure
+    runs: |
+      ./configure \
+         --host=${{host.triplet.gnu}} \
+         --target=${{host.triplet.gnu}} \
+         --prefix=/usr \
+         --datadir=/usr/share \
+         --with-external-libmspack
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: false
```

Comment:
As the source is on GitHub we can track the tags https://github.com/kyz/libmspack/tags